ql/src/test/org/apache/hadoop/hive/ql/exec/repl/TestReplDumpTask.java

```diff
@@ -127,13 +128,15 @@ public void removeDBPropertyToPreventRenameWhenBootstrapDumpOfTableFails() throw
       private int tableDumpCount = 0;
 
       @Override
-      void dumpTable(String dbName, String tblName, String validTxnList, Path dbRoot, Path replDataDir,
-                     long lastReplId, Hive hiveDb, HiveWrapper.Tuple<Table> tuple)
+      List<EximUtil.ReplPathMapping> dumpTable(String dbName, String tblName, String validTxnList, Path dbRoot,
+                                               Path replDataDir, long lastReplId, Hive hiveDb,
+                                               HiveWrapper.Tuple<Table> tuple)
           throws Exception {
         tableDumpCount++;
         if (tableDumpCount > 1) {
           throw new TestException();
         }
+        return Collections.EMPTY_LIST;
```

Comment:
better to use Collections.emptyList() here and other places, the generic cast would  be provided by this.

Reply:
done